ajax/libs/PreloadJS/package.json

```diff
@@ -2,22 +2,32 @@
   "name": "PreloadJS",
   "filename": "preloadjs.min.js",
   "version": "0.6.0",
-  "description": "A Javascript library that lets you manage and co-ordinate the loading of assets.",
-  "homepage": "http://www.createjs.com/#!/PreloadJS",
+  "description": "PreloadJS makes it easy to preload your assets: images, sounds, JavaScript, fonts, JSON, and text data",
   "keywords": [
-    "load",
-    "image",
-    "assets",
-    "preload"
+    "createjs",
+    "preloadjs",
+    "gskinner",
+    "preload",
+    "preloading",
+    "progress",
+    "XHR",
+    "javascript",
+    "html5"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/CreateJS/PreloadJS.git"
+    "url": "git+https://github.com/CreateJS/PreloadJS.git"
   },
   "license": "MIT",
-  "authors": [
-    "gskinner",
-    "lannymcnie",
-    "wdamien"
+  "author": "gskinner.com, inc.",
+  "homepage": "https://createjs.com/preloadjs/",
+  "npmName": "preloadjs",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
Need to modify this to `**/!(README.md)` ( or `**/!(*.md)` if you could put any other markdown file under `dist` in the future) to exclude `README.md`

Reply:
@wdamien Please help modify the pattern to `**/!(*.md)`
Thank you.
(Except for this part, I think this PR is GOOD!)